install/index.adoc

```diff
@@ -331,19 +325,19 @@ at the beginning of the installation process to ensure that these settings are
 applied before attempting to pull any of the required images.
 
 [[planning-installation-types-service-names]]
-=== systemd Service Names
+=== systemd service names
 
 The installation process creates relevant *systemd* units which can be used to
 start, stop, and poll services using normal *systemctl* commands. For system
 container installations, these unit names match those of an RPM installation.
 
 [NOTE]
 ====
-The *etcd* package is slated to be removed from RHEL Atomic Host in the future.
+The *etcd* package will be removed from RHEL Atomic Host in the future.
```

Comment:
I'd remove this note, its no longer relevant in so much that now etcd runs as a pod on atomic host and the package was never removed and won't be prior to 4.0.